dev/Dockerfile

```diff
@@ -32,17 +32,19 @@ ENV NVM_DIR /root/.nvm
 
 RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
 
-# these are the stable versions as of July 2021
-# cant use lts alias due to all sorts of Dockerfile limitations.
+# these are the stable versions as of November 2021
+# can't use lts alias due to all sorts of Dockerfile limitations.
 RUN source $NVM_DIR/nvm.sh \
-    && nvm install v14.17.5 \
-    && nvm install v12.22.5 \
+    && nvm install v14.18.1 \
```

Comment:
maybe sort the list

Reply:
or is 14 kept on top because it is the base version for the image?

Reply:
> or is 14 kept on top because it is the base version for the image?

Yep ;)

(14 to 16 has npm change with side effects. 14 stays "base" for node agent until April next year. Customer version "tail" is long....).